COPYING

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2012-2021 Juan Luis Cano Rodríguez and the poliastro development team
+Copyright (c) 2012-2023 Juan Luis Cano Rodríguez and the poliastro development team
```

Comment:
```suggestion
Copyright (c) 2012 Juan Luis Cano Rodríguez, Jorge Martínez Garrido, and the poliastro development team
```

I discovered some time ago that we don't need to update the final year, the initial one is what counts.

Also, time to include you in the authors list!